rpmvenv/extensions/core.py

```diff
@@ -85,11 +93,17 @@ def generate(config, spec):
         source = config.core.source
         buildroot = config.core.buildroot
         buildarch = config.core.buildarch
+        requires = config.core.requires
+        provides = config.core.provides
 
         spec.tags['Name'] = name
         spec.tags['Version'] = version
         spec.tags['Release'] = release
         spec.tags['BuildRoot'] = buildroot
+        spec.tags['Requires'] = requires
+        spec.tags['Provides'] = provides
```

Comment:
Since these are optional additions to the configuration file they need to be optionally included in the SPEC. For example:

```python
if requires:

    spec.tags['Requires'] = requires
```

If you expand this diff below just a bit there are some examples of the other optional bits.